src/utils/utils.ts

```diff
@@ -150,3 +150,25 @@ export const sanitizeStringForAttribute = (str: string) => {
   const sanitizedStr = str.replace(/[^a-z0-9]/gi, "-").toLowerCase();
   return sanitizedStr;
 };
+
+/** Provides a random image to use as a placeholder. */
+// export interface GetPlaceholderImageProps {
+//   size?: "small" | "medium" | "large";
+// }
+export const getPlaceholderImage = (size: string = "medium") => {
```

Comment:
@EdwinGuzman I added this function and it works well.  However, I first tried to set it up using an interface and ran into issues where typescript was saying that the at least one arg was required.  To see this fucntion in action, take a look at the `Hero` component.

Reply:
I'm not sure how `GetPlaceholderImageProps` was assigned, but because this is a single argument value and not an object, you can't do something like `getPlaceholderImage = (size: GetPlaceholderImageProps = "medium") => {`.

What you can do is make this a type instead:
```
export type GetPlaceholderImageProps = "small" | "medium" | "large";
export const getPlaceholderImage = (
  size: GetPlaceholderImageProps = "medium"
) => {
```